config/samples/druid_v1alpha1_etcd.yaml

```diff
@@ -74,5 +74,5 @@ spec:
 
   replicas: 1
 # priorityClassName: foo
-  storageClass: standard
+  storageClass: default
```

Comment:
Shouldn't we use `gardener.cloud-fast` here?

Reply:
`gardener.cloud-fast` is a special storage class created and used by Gardener, that too only for seed clusters by default. Since etcd-druid is also meant to be consumed by the larger community and not just by Gardener, it's important to use standard storageClass in the sample Etcd resource, which is `default`.

Reply:
Thanks for the clarification